config/localTemplate.go

```diff
@@ -771,9 +771,13 @@ func (cfg *Local) ResolveLogPaths(rootDir string) (liveLog, archive string) {
 	return liveLog, archive
 }
 
+type logger interface {
+	Infof(format string, args ...interface{})
+}
```

Comment:
Unless there's some sort of circular dependency, you should be able to use the existing testing logger.

Reply:
I expect that advice isn't typically applied to cross cutting dependencies like logging. Our mistake is in customizing the logging library in algod instead of having a pure dependency.

Still. Circular dependencies are annoying and this is a good workaround.